spec/config/routes.pending.yml

```diff
@@ -268,6 +268,16 @@ ConfigurationProfileController:
 - index
 - quick_search
 - reload
+- adv_search_button
+- adv_search_clear
+- adv_search_load_choice
+- adv_search_name_typed
+- adv_search_toggle
+- search_clear
+- exp_button
+- exp_changed
+- exp_token_pressed
+- save_default_search
```

Comment:
Are these routes marked pending elsewhere in this file?  If so, this is fine, since it's a pattern.  If not, that means that you need to added proper RBAC to the various endpoints.

Reply:
> Are these routes marked pending elsewhere in this file? If so, this is fine, since it's a pattern. If not, that means that you need to added proper RBAC to the various endpoints.

Yes some of the other controllers have them marked pending as well such as https://github.com/ManageIQ/manageiq-ui-classic/blob/3926ad9c68b7c3e32def6d5dda413bbcaa546322/spec/config/routes.pending.yml#L607-L621